test/model/schemas/schema_v1.dart

```diff
@@ -1,6 +1,6 @@
+// dart format width=80
 // GENERATED CODE, DO NOT EDIT BY HAND.
 // ignore_for_file: type=lint
-//@dart=2.12
 import 'package:drift/drift.dart';
```

Comment:
Similarly, awkward that these `schema_vN.dart` files change, but again the change is a no-op (and this time just `git log -p` is enough to show that).